docs/releases/v2019.1.1.rst

```diff
@@ -0,0 +1,60 @@
+Gluon 2019.1.1
+##############
+
+Bugfixes
+********
+
+* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)
+
+* Uses correct ath10k firmware for OCEDO Koala. (`#1838 <https://github.com/freifunk-gluon/gluon/pull/1838>`_)
+
+* Fixes various batman-adv bugs with backports from 2019.4 and 2019.5 by updating the openwrt-routing packages feed.
```

Comment:
As v2018.2.4.

Reply:
i took the text adjustments from v2018.2.4